modules/logging-architecture-overview.adoc

```diff
@@ -24,6 +34,4 @@ The {logging-title} collects container logs and node logs. These are categorized
 
 * `audit` - Logs generated by auditd, the node audit system, which are stored in the */var/log/audit/audit.log* file, and logs from the `auditd`, `kube-apiserver`, `openshift-apiserver` projects, as well as the `ovn` project if enabled.
 
-The logging collector is a daemonset that deploys pods to each {product-title} node. System and infrastructure logs are generated by journald log messages from the operating system, the container runtime, and {product-title}.
-
-Container logs are generated by containers running in pods running on the cluster. Each container generates a separate log stream. The collector collects the logs from these sources and forwards them internally or externally as configured in the `ClusterLogForwarder` custom resource.
+System and infrastructure logs are generated by journald log messages from the operating system, the container runtime, and {product-title}.
```

Comment:
Infrastructure from a logging perspective are:

* journald message from the node (OS logs)
* container logs from the infra NS (default, kube*,openshift*). 

We may not want to explicitly define the list of NS but we should at least note that some "infrastructure" logs are container workloads that come from "infrastructure" namespaces.

Reply:
Decided to make some additional updates to the infra bit based on your and @JoaoBraveCoding's suggestions, thank you 😄 let me know if that's better 🙏🏻